lib/laa_crime_schemas/structs/saving.rb

```diff
@@ -4,14 +4,14 @@ module LaaCrimeSchemas
   module Structs
     class Saving < Base
       attribute :saving_type, Types::SavingType
-      attribute :provider_name, Types::String
+      attribute? :provider_name, Types::String.optional
       attribute :account_balance, Types::PenceSterling
```

Comment:
Is this meant to be optional too?

Reply:
Yes, the json-schema makes sure this is required on submission, but can be nil in draft.